set.mm

```diff
@@ -634971,10 +634988,57 @@ fixed reference functional determined by this vector (corresponding to
       UCUNVUKUOUIULVUNYRABEJHUVCUVIFYOORUAUDUNVUMUOUJUMVUNYRWQYS $.
   $}
 
+  ${
+    ringgrpd.1 $e |- ( ph -> R e. Ring ) $.
+    $( A ring is a group.  (Contributed by SN, 16-May-2024.) $)
+    ringgrpd $p |- ( ph -> R e. Grp ) $=
+      ( crg wcel cgrp ringgrp syl ) ABDEBFECBGH $.
+  $}
+
+  ${
+    drngringd.1 $e |- ( ph -> R e. DivRing ) $.
+    $( A division ring is a ring.  (Contributed by SN, 16-May-2024.) $)
+    drngringd $p |- ( ph -> R e. Ring ) $=
+      ( cdr wcel crg drngring syl ) ABDEBFECBGH $.
+
+    $( A division ring is a group.  (Contributed by SN, 16-May-2024.) $)
+    drnggrpd $p |- ( ph -> R e. Grp ) $=
+      ( drngringd ringgrpd ) ABABCDE $.
+  $}
+
+  ${
+    crngringd.1 $e |- ( ph -> R e. CRing ) $.
+    $( A commutative ring is a ring.  (Contributed by SN, 16-May-2024.) $)
+    crngringd $p |- ( ph -> R e. Ring ) $=
+      ( ccrg wcel crg crngring syl ) ABDEBFECBGH $.
+
+    $( A commutative ring is a group.  (Contributed by SN, 16-May-2024.) $)
+    crnggrpd $p |- ( ph -> R e. Grp ) $=
+      ( crngringd ringgrpd ) ABABCDE $.
+  $}
+
+  ${
+    lmodgrpd.1 $e |- ( ph -> W e. LMod ) $.
+    $( A left module is a group.  (Contributed by SN, 16-May-2024.) $)
+    lmodgrpd $p |- ( ph -> W e. Grp ) $=
+      ( clmod wcel cgrp lmodgrp syl ) ABDEBFECBGH $.
+  $}
+
   $( A left vector is a group.  (Contributed by Steven Nguyen, 28-May-2023.) $)
   lvecgrp $p |- ( W e. LVec -> W e. Grp ) $=
     ( clvec wcel clmod cgrp lveclmod lmodgrp syl ) ABCADCAECAFAGH $.
 
+  ${
+    lveclmodd.1 $e |- ( ph -> W e. LVec ) $.
+    $( A left vector is a left module.  (Contributed by SN, 16-May-2024.) $)
```

Comment:
```suggestion
    $( A vector space is a left module.  (Contributed by SN, 16-May-2024.) $)
```
or just "module", since we only have left modules in set.mm.